pinot-common/src/main/java/com/linkedin/pinot/common/data/StarTreeIndexSpec.java

```diff
@@ -96,4 +117,15 @@ public void setExcludeSkipMaterializationDimensionsForStarTreeIndex(
   public String toString() {
     return ToStringBuilder.reflectionToString(this, ToStringStyle.SHORT_PREFIX_STYLE);
   }
+
+  public String toJsonString() {
+    Map<String, Object> map = new HashMap<>();
```

Comment:
Use ObjectMapper to serialize (and de-serialize) to (from) JSON.

Reply:
done